modules/private.php

```diff
@@ -0,0 +1,298 @@
+<?php
+/**
+ * Module Name: Private site
+ * Module Description: Make your site only visible to you and users you approve.
+ * Sort Order: 9
+ * First Introduced: ?
+ * Requires Connection: No
+ * Auto Activate: No
+ * Module Tags: Private
+ * Feature: Traffic
+ * Additional Search Queries: private, sandbox, launch, unlaunched
+ *
+ * @package Jetpack
+ */
+
+class Jetpack_Private {
+	static function init() {
+		add_action( 'parse_request', array( __CLASS__, 'privatize_blog' ), 100 );
+		add_action( 'jetpack_activate_module_private', array( __CLASS__, 'make_blog_private' ) );
+		add_action( 'jetpack_deactivate_module_private', array( __CLASS__, 'make_blog_public' ) );
+		add_action( 'login_init', array( __CLASS__, 'privatize_blog_maybe_mask_blog_name' ) );
+		add_filter( 'preprocess_comment', array( __CLASS__, 'privatize_blog_comments' ) );
+		add_action( 'blog_privacy_selector', array( __CLASS__, 'privatize_blog_priv_selector' ) );
+		add_filter( 'robots_txt', array( __CLASS__, 'private_robots_txt' ) );
+		add_filter( 'privacy_on_link_title', array( __CLASS__, 'privatize_privacy_on_link_title' ) );
```

Comment:
This is not very helpful here because the message is only displayed when `blog_public` is set to `0`, and in our case it is set to `-1`:

https://github.com/WordPress/WordPress/blob/91da29d9afaa664eb84e1261ebb916b18a362aa9/wp-admin/includes/dashboard.php#L331

Reply:
Good point. I removed these filters and added a new one for `update_right_now_text`, which adds a message in the same place.